transforms/tabular/polus-csv-merger-plugin/src/main.py

```diff
@@ -57,15 +75,33 @@
         for key in out_files.keys():
             
             outPath = str(Path(outDir).joinpath('merged_{}.csv'.format(count)).absolute())
+            
             count += 1
             
             inp_files = [open(f) for f in out_files[key]]
             
+            # # Write Merged CSV
```

Comment:
If we are using feather for this, we should completely rewrite this section since we no longer need to write the csv file manually